### PR TITLE
ddns-scripts: Append ddns script for cnkuai.cn

### PR DESCRIPTION
**Pull request by AuroraRAS**
Maintainer: Community
Compile tested: (Atheros AR7161 rev 2, all, OpenWrt 18.06.4 r7808-ef686b7292)
Run tested: (Atheros AR7161 rev 2, NETGEAR WNDR3800, OpenWrt 18.06.4 r7808-ef686b7292, tests done)

Description:
Append ddns script support for a low-cost domain name service provider in China.

### CODE REVIEW

**Comment by thess** [comment]
@AuroraRAS - Sorry, I am not the maintainer of this package. I removed my name from this PR.. Others will review it.

**Comment by AuroraRAS** [comment]
> @AuroraRAS - Sorry, I am not the maintainer of this package. I removed my name from this PR.. Others will review it.

a copy error, I'm sorry about that.

**Comment by neheb** [comment]
ping @dibdot

**Comment by dibdot** [comment]
@AuroraRAS Just a small formal feedback, as I don't use this package:

- please sign-off your work with your real name (no abbreviations)
- please raise the 'PKG_RELEASE' in the makefile as well
- please use '$(...)' notation instead of legacy backticked '`...`' in your script
- please protect strings with double quotes, e.g. `echo "$(hexdump ...)"` n your script

Thanks!

File: net/ddns-scripts/files/update_cnkuai_cn.sh [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# inside url we need domain, username and password
```
Comment (neheb) [inline net/ddns-scripts/files/update_cnkuai_cn.sh]: Please run through shellcheck.

**Comment by AuroraRAS** [comment]
> @AuroraRAS Just a small formal feedback, as I don't use this package:
> 
> * please sign-off your work with your real name (no abbreviations)
> * please raise the 'PKG_RELEASE' in the makefile as well
> * please use '$(...)' notation instead of legacy backticked '`...`' in your script
> * please protect strings with double quotes, e.g. `echo "$(hexdump ...)"` in your script
> 
> Thanks!

Thanks for your feedback, I'll reopen the request after I fix them.

File: net/ddns-scripts/files/update_cnkuai_cn.sh (hunk 1 above)
Comment (AuroraRAS) [reply net/ddns-scripts/files/update_cnkuai_cn.sh]: > Please run through shellcheck.

That's awesome tool. You let me know it, thank you.

**Comment by dibdot** [comment]
@AuroraRAS @neheb If you're going the (s)hellcheck way please note, that in this case all "local" related errors/warnings are incorrect, as this script is sourced in another shell function - please ignore such errors.

**Comment by AuroraRAS** [comment]
> @AuroraRAS @neheb If you're going the (s)hellcheck way please note, that in this case all "local" related errors/warnings are incorrect, as this script is sourced in another shell function - please ignore such errors.

I noticed this detail. thanks.

**Comment by AuroraRAS** [comment]
I cannot reopen the pull again because force-pushed.
a new pull request is created at https://github.com/openwrt/packages/pull/11624

**Pull request closed without merging**